### ISSUE TITLE
Dockerfile for running locally?

### ISSUE THREAD

**Issue by kevinlinxc** [carbon-app/carbon#1529]
I would like to use carbon.now.sh at work, but I don't want to send sensitive data to your servers. I noticed you can run locally with yarn and npm, I'm wondering if you could set up a Dockerfile that runs it locally without needing to install these tools?

**Comment by knzcx** [comment]
Yes, a Dockerfile would be helpful or instructions on how to start the project locally. But I can't find instructions anywhere.

**Comment by jeremych1000** [comment]
Unfortunately it seems like the devs don't want this to be easy. As referenced above - you can copy one from https://github.com/carbon-app/carbon/blob/c61e0569f3ed2231289a8f869b76198d83cb0839/Dockerfile
